.taskcluster.yml

```diff
@@ -2,19 +2,22 @@
 version: 1
 reporting: checks-v1
 policy:
-    pullRequests: public
+    pullRequests: collaborators
 tasks:
     - $let:
           taskgraph:
               branch: taskgraph
-              revision: c78d65237c6ed064b9cbb3cd063b22e9e052a06a
+              revision: 7dde9ce7740068d7b73218976343a28fc99be847
           template:
-              repo: https://github.com/mozilla-extensions/xpi-template
-              branch: master
+              #XXX uncomment to merge
+              #repo: https://github.com/mozilla-extensions/xpi-template
+              #branch: master
+              repo: https://github.com/escapewindow/xpi-template
+              branch: private-ci
           trustDomain: xpi
           # XXX for private repos, uncomment this line, and comment out the next
-          # githubCloneSecret: project/xpi/xpi-github-clone-ssh
-          githubCloneSecret: ""
+          #xpiSshSecretName: project/xpi/xpi-github-clone-ssh
```

Comment:
I think it would be better if this were a boolean, rather than something to comment or uncomment.